pyramid/pyramid.go

```diff
@@ -1,5 +1,7 @@
 package pyramid
 
+//go:generate mockgen -source=pyramid.go -destination=mock/pyramid.go -package=mock
```

Comment:
Great!

Reply:
Thanks I had to change the FS interface in order to make the mockgen to work